PixelToolkit.py

```diff
@@ -38,6 +38,13 @@
     wordlist_gen.add_argument("--min", type=int, help="Minimum keyword length", default=1)
     wordlist_gen.add_argument("--max", type=int, help="Maximum keyword length", default=100)
 
+    hash = subparsers.add_parser("hash", help="Text hashing")
+    hash.add_argument("--text", "-t", required=True, help="File or text to be hashed. Files will be hashed line by line.")
```

Comment:
`--text` should not accept files. `--file` should do that (I did a similar thing in wordlist generator (user can choose to either use a URL or a File)). Also `Files will be hashed line by line` is that still true?

Reply:
No. It's some leftover from before.
I'll change it to input (what do you think?).

Reply:
I still think splitting it into --text and --file is better because when the user types a path that doesn't exist (eg makes a typo) the code will asume that was a string and will print the hash of it which is unexpected in my opinion